File: 1주차/선주/치킨쿠폰.py
def solution(chicken):
    answer = -1
    coupone = chicken/10
    service = 0
    while (coupone >0):
        service += coupone
        coupone //=10   
    
    return service # 실행한 결괏값 119.1이 기댓값 120과 다릅니다.

File: 1주차/성실/치킨 쿠폰.java
class Solution {
    public int solution(int chicken) {
        int coupon = chicken;
        int service = 0;
        
        while(true) {
            service += coupon / 10;
            coupon = coupon / 10 + coupon % 10;
            
            if (coupon < 10) {
                break;
            }
        }
        
        
        //int answer = -1;
        //return answer;
        return service;
    }
}
